test/unit/yarn-workspaces/output.js

```diff
@@ -1,6 +1,8 @@
 [
+  "package.json",
   "test/unit/yarn-workspaces/input.js",
   "test/unit/yarn-workspaces/node_modules/x",
+  "test/unit/yarn-workspaces/node_modules/x/package.json",
```

Comment:
@guybedford I think this file was incorrectly emitted since this line is a symlink pointing to the last line in the output here. See PR #135 for the fix.